packages/lib/services/e2ee/EncryptionService.ts

```diff
@@ -278,8 +299,9 @@ export default class EncryptionService {
 		if (!key) throw new Error('Encryption key is required');
 
 		const sjcl = shim.sjclModule;
+		const crypto = shim.crypto;
 
-		const handlers: Record<EncryptionMethod, ()=> string> = {
+		const handlers: Record<EncryptionMethod, ()=> Promise<string>> = {
```

Comment:
<details><summary>🔵 Optional/style: Extract <code>()=> Promise&lt;string&gt;</code> to a separate type.</summary>

```suggestion
		type EncryptionMethodHandler = ()=> Promise<string>;
		const handlers: Record<EncryptionMethod, EncryptionMethodHandler> = {
```

</details>

Reply:
Finished in commit 91f9c9b81144747532a1b92c19bbb23ef30ca654.